packages/contentstack/package.json

```diff
@@ -34,10 +34,10 @@
     "@contentstack/cli-cm-migrate-rte": "~1.4.15",
     "@contentstack/cli-cm-seed": "~1.7.1",
     "@contentstack/cli-command": "~1.2.17",
-    "@contentstack/cli-config": "~1.5.1",
+    "@contentstack/cli-config": "~1.6.0",
     "@contentstack/cli-launch": "~1.0.15",
     "@contentstack/cli-migration": "~1.4.2",
```

Comment:
Bump the core package

Reply:
@shafeeqd959 will bump the core package version before merging the gcp branch to dev

Reply:
@shafeeqd959 will bump the core package version before merging the gcp branch to dev